tests/Unit/Infrastructure/Auth/RoleAccessTest.php

```diff
@@ -55,23 +55,60 @@ public function testReturnsNothingIfCheckSucceededAndUserHasAdminPermission()
         $auth = Mockery::mock(Authentication::class);
         $auth->shouldReceive('check')->andReturn(true);
         $auth->shouldReceive('user')->andReturn($user);
-        $this->swap(Authentication::class, $auth);
 
         //The middleware doesn't do anything if the user is an admin, so it returns null (void)
-        $this->assertNull(RoleAccess::userHasAccess($this->app, 'admin'));
+        $this->assertNull(RoleAccess::userHasAccess($auth, 'admin'));
     }
 
     public function testReviewerCantGetToAdminPages()
```

Comment:
The entire test just duplicates `testReturnsFalseIfCheckSucceededButUserHasNoAdminPermission()`, can we remove this test here?